engine/src/main/java/se/swedenconnect/signservice/engine/DefaultSignRequestMessageVerifier.java

```diff
@@ -110,8 +124,8 @@ public void verifyMessage(final SignRequestMessage signRequestMessage, final Eng
           throw new SignServiceErrorException(new SignServiceError(SignServiceErrorCode.REQUEST_EXPIRED, null, msg));
         }
       }
-      else if ((now.toEpochMilli() - issuedAt.toEpochMilli())
-          > (this.maxMessageAge.toMillis() + this.allowedClockSkew.toMillis())) {
+      else if ((now.toEpochMilli() - issuedAt.toEpochMilli()) > (this.maxMessageAge.toMillis()
+          + this.allowedClockSkew.toMillis())) {
 
```

Comment:
I prefer .isBefore and .isAfter functions of Instant. It makes the logic much more clear and may help avoid future bugs. (See my new checkSigningTime() function in AbstractTBSDataProcessor. Such function could be made available using a global utility class if generalised?